common/BUILD

```diff
@@ -12,6 +12,29 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "argparse",
+    srcs = ["argparse.cpp"],
+    hdrs = ["argparse.h"],
```

Comment:
I believe our style is to use underscores to separate words in filename (arg_parse.h, rather than argparse.h)

Reply:
Given the naming discussion and moving this to `CommandLine`, I'll rename the files and library to use `command_line` as the base instead of `argparse` before I land this. I'm not doing that just yet to keep comment threads in the code.